spec/features/bulk_updates/bulk_updates_location_keywords_spec.rb

```diff
@@ -394,15 +395,16 @@
         before do
           visit collection_path(@clear_all_and_replace_ingest_response['concept-id'])
 
-          find('.tab-label', text: 'Additional Information').click
         end
 
         it 'displays the updated keywords' do
-          within '.location-keywords-preview' do
+          within '#metadata-preview' do
             expect(page).to have_no_content('GEOGRAPHIC REGION ARCTIC', normalize_ws: true)
```

Comment:
I think the keywords to check against are GEOGRAPHIC REGION, ARCTIC OCEAN, ATLANTIC OCEAN